tests/unit/components/record/MediaPresentation.spec.js

```diff
@@ -95,7 +95,7 @@ describe('components/record/MediaPresentation', () => {
 
         wrapper.setProps(props);
 
-        wrapper.vm.isOEmbed.should.be.true;
+        wrapper.vm.isOEmbed.should.be.false;
```

Comment:
instead of testing that youtube is false, let's test for an actual oEmbed provider we do support.